anchor/src/environment.rs

```diff
@@ -8,6 +8,8 @@ use task_executor::{ShutdownReason, TaskExecutor};
 use tokio::runtime::{Builder as RuntimeBuilder, Runtime};
 use tracing::{error, info, warn};
 use tracing_subscriber::EnvFilter;
+
+#[cfg(target_family = "unix")]
```

Comment:
Won't this break the application for non-unix machines? Usually we'd import non-unix crates that are compatible. 

I've not tested anchor on non-unix, but if this currently doesn't work on non-unix, we probably should address it correctly.

Reply:
This was a mistake to include this change in this PR. 
The fix is done in https://github.com/sigp/anchor/pull/68